votingpool/db.go

```diff
@@ -70,7 +139,7 @@ func putPool(tx walletdb.Tx, votingPoolID []byte) error {
 // loadAllSeries returns a map of all the series stored inside a voting pool
 // bucket, keyed by id.
 func loadAllSeries(tx walletdb.Tx, votingPoolID []byte) (map[uint32]*dbSeriesRow, error) {
```

Comment:
I know this already exists, but just adding this here for discussion.  The same thing is being done over in `waddrmgr` where there is a load all type of function.  Unfortunately, this approach will have scalability issues.

We are considering adding a `Cursor` to walletdb potentially along with an `AddressIterator` to waddrmgr.  Do you have any suggestions and or issues in regards to this?  I think it would definitely apply here as I imagine the number of series can get huge over time.